receipt_upload/receipt_upload/label_validation/llm_validator.py

```
@@ -0,0 +1,782 @@
+class LabelValidationResponse(BaseModel):
+    """Structured response containing all label validation decisions."""
+
+    decisions: List[LabelDecision] = Field(
+        description="List of validation decisions, one per pending label"
+    )
+
+
+def convert_structured_response(
+    response: LabelValidationResponse,
+    pending_labels: List[Dict[str, Any]],
+) -> List[LLMValidationResult]:
```

Ruff (0.14.10)

Undefined name `LLMValidationResult`

(F821)